src/components/connectWallet/ConnectWallet.vue

```diff
@@ -0,0 +1,275 @@
+<template>
+  <div class="modal-backdrop">
+    <div class="modal">
+      <img class="wallet_gif" src="../../assets/icons/Wallet.png" />
+
+      <div class="title">
+        <div class="header connect_wallet_header">Connect your wallet</div>
+        <div class="connect_wallet_info">Start using Yarn Synths with your favourite wallet</div>
+      </div>
+
+      <div class="connect_wallet_container">
+        <div @click="$emit('connect', 'injected')" class="wallet_box">
+          <img src="../../assets/icons/metamask.svg" />
+          <div class="wallet_name" >Metamask</div>
+        </div>
+        <div @click="$emit('connect', 'walletconnect')" class="wallet_box">
+          <img src="../../assets/icons/walletconnect.png" />
+          <div class="wallet_name" >WalletConnect &nbsp;</div>
+        </div>
+        <div @click="$emit('connect', 'fortmatic')" class="wallet_box">
+          <img src="../../assets/icons/formatic.png" />
+          <div class="wallet_name" @click="$emit('connect', 'fortmatic')">Fortmatic</div>
+        </div>
+        <div @click="$emit('connect', 'portis')" class="wallet_box">
+          <img src="../../assets/icons/portis.png" />
+          <div class="wallet_name">Portis</div>
+        </div>
+      </div>
+      <div class="learn_more_box">
+        Not sure how connect wallet works?
+        <span class="t_learn_more"
+          >Learn More &nbsp;<img src="../../assets/icons/externalLink.svg"
+        /></span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script >
+
+
+
+export default {
+  name: "ConnectWallet",
+
+  methods: {
+    close() {
+      console.log("sadfasdf");
+      this.$emit("close")
+    },
+    saveWalletType() {
+      this.$emit("saveWalletType")
+    },
+ 
+    
+  },
+}
+</script>
+<style>
+.modal-backdrop {
```

Comment:
all this should be tailwind inline and not like done like that, follow how the project is strucutred and check the other files to see how this handled

Reply:
@punk1004 still unresolved, for any colors also use inline